src/platform/Tizen/ConfigurationManagerImpl.cpp

```diff
@@ -0,0 +1,79 @@
+/*
+ *
+ *    Copyright (c) 2020 Project CHIP Authors
+ *    Copyright (c) 2018 Nest Labs, Inc.
+ *    All rights reserved.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+/**
+ *    @file
+ *          Provides the implementation of the Device Layer ConfigurationManager object
+ *          for Tizen platforms.
+ */
+
+#include <platform/internal/CHIPDeviceLayerInternal.h>
+
+#include <core/CHIPVendorIdentifiers.hpp>
+#include <platform/ConfigurationManager.h>
+#include <platform/Tizen/PosixConfig.h>
+#include <platform/internal/GenericConfigurationManagerImpl.cpp>
+#include <support/CodeUtils.h>
+#include <support/logging/CHIPLogging.h>
+
+namespace chip {
+namespace DeviceLayer {
+
+using namespace ::chip::DeviceLayer::Internal;
+
+/** Singleton instance of the ConfigurationManager implementation object.
+ */
+ConfigurationManagerImpl ConfigurationManagerImpl::sInstance;
+
+CHIP_ERROR ConfigurationManagerImpl::_Init(void)
+{
+    CHIP_ERROR err = CHIP_NO_ERROR;
+
+    // Initialize the generic implementation base class.
+    err = Internal::GenericConfigurationManagerImpl<ConfigurationManagerImpl>::_Init();
+    SuccessOrExit(err);
+
+exit:
+    return err;
```

Comment:
replace this entire method with:

```cpp
return Internal::GenericConfigurationManagerImpl<ConfigurationManagerImpl>::_Init();
```

The 'goto exit' pattern here went out of control.

Reply:
Done.